src/main/js/locale/index.js

```diff
@@ -49,6 +53,8 @@ const LOCALE = {
     es_ES: es,
     fr_FR: fr,
     pt_BR: br,
-    ja_JP: jp
+    ja_JP: jp,
+    nl_NL: nl,
```

Comment:
* Since this is a new functionality, can you update the title and commit message to `New`?
* Can you update the documentation? https://github.com/cerner/carbon-graphs/tree/master/docs#locale-1